standup/status/views.py

```diff
@@ -159,21 +156,20 @@ def form_invalid(self, form):
         return super().form_invalid(form)
 
 
-def is_auth0_configured():
-    return (
-        settings.AUTH0_CLIENT_ID and
-        settings.AUTH0_CLIENT_SECRET and
-        settings.AUTH0_DOMAIN and
-        settings.AUTH0_CALLBACK_URL
-    )
-
-
 class LoginView(TemplateView):
     template_name = 'users/login.html'
 
     def get_context_data(self, **kwargs):
         ctx = super().get_context_data(**kwargs)
         ctx['auth0_configured'] = is_auth0_configured()
+
+        ctx['auth0_login_url'] = settings.AUTH0_LOGIN_URL.format(
+            AUTH0_DOMAIN=settings.AUTH0_DOMAIN,
+            AUTH0_CLIENT_ID=settings.AUTH0_CLIENT_ID,
+            AUTH0_CALLBACK_URL=settings.AUTH0_CALLBACK_URL,
+            # FIXME(willkg): This should be a token that ties both ends.
+            STATE='foo',
```

Comment:
Should we fix this before prod? Is it just some string that should be unique per instance? A setting?